search/search-tool/tool/pom.xml

```diff
@@ -100,6 +100,11 @@
         <artifactId>commons-lang3</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.elasticsearch</groupId>
```

Comment:
Is this dependency required? It doesn't look like it's used anywhere in the tool.